application/console/console.rb

```diff
@@ -0,0 +1,99 @@
+# frozen_string_literal: true
+
+class Console < ConsoleHelper
```

Comment:
Do not inherit a class from the helper. If the helper contains common helper methods, then include it as a module.

Reply:
created ConsoleHelper as module and included it in CardsConsole and Console classes.